bin/download-import-tag

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+$(curl "https://hg.mozilla.org/mozilla-unified/archive/$1.tar.gz" > tmp/$1.tar.gz) && echo Importing with bin/firefox-schema-import tmp/$1.tar.gz && bin/firefox-schema-import tmp/$1.tar.gz
```

Comment:
Can we make use of newlines to make this easier to read and less prone to horizontal scrolling?

Something like

```bash
#!/usr/bin/env bash

# Download a mozilla-unified tag into a temporary file
curl "https://hg.mozilla.org/mozilla-unified/archive/$1.tar.gz" > tmp/$1.tar.gz

echo Importing with bin/firefox-schema-import tmp/$1.tar.gz

# Now import the schema from that tag
bin/firefox-schema-import tmp/$1.tar.gz 
```

should work too (warning, untested)